focuspoints.lrdevplugin/DefaultPointRenderer.lua

```diff
@@ -128,6 +135,10 @@ function DefaultPointRenderer.getAutoFocusPoint(metaData)
     focusPointUsed = ExifUtils.findValue( metaData, DefaultPointRenderer.metaPrimaryAFPoint )
   end
 
+  if "(none)" == focusPointUsed then
```

Comment:
i'll play around this with some after work. Question for you, any idea how universal "(none)" will be? Do you think that string will hold up for pentax or canons?